src/easy-popup.js

```diff
@@ -302,6 +302,8 @@
 
     // Get popup object by ID
     EasyPopup.get = id => window.EasyPopupData.get(id);
+    console.log("load")
```

Comment:
Could you remove this console log, please?

Reply:
Sure